activerecord/lib/active_record/connection_adapters/postgresql/oid/serial.rb

```diff
@@ -0,0 +1,10 @@
+module ActiveRecord
+  module ConnectionAdapters
+    module PostgreSQL
+      module OID # :nodoc:
+        class Serial < Integer # :nodoc:
+        end
+      end
+    end
+  end
+end
```

Comment:
Ok.  It was hard to figure out all the parts required to add a column type.  I never tested it without this code.